test/typescript/response-generators/message.js

```diff
@@ -17,7 +17,11 @@ async function getMessage() {
 	const channel = authClient.channel('messaging', `poppins-${uuidv4()}`);
 	await channel.watch();
 	const { message } = await channel.sendMessage({ text: `Test message` });
-
+	await channel.sendMessage({
+		text: 'Hey, I am replying to a message!',
+		parent_id: message.id,
+		show_in_channel: false,
+	});
```

Comment:
Just to ensure, any thread related stuff gets type-checked.

Reply:
not actually sure how this test runs but how is it ensuring the type-check is run? is show_in_channel not defaulted to false already?